### DISCUSSION TITLE
新版本的 Docker Image 拉不下来有无解决办法

### DISCUSSION THREAD

**Discussion by zhangleijun111** [lobehub/lobehub#2852] [Self Hosting] [answered]
### 📦 部署环境

- [ ] Official
- [ ] Official Preview
- [ ] Vercel / Zeabur / Sealos
- [X] Docker
- [ ] Other

### 📌 软件版本

7.9

### 💻 系统环境

- [ ] Windows
- [ ] macOS
- [ ] Ubuntu
- [X] Other Linux
- [ ] iOS
- [ ] Android
- [ ] Other

### 🌐 浏览器

- [X] Chrome
- [ ] Edge
- [ ] Safari
- [ ] Firefox
- [ ] Other

### 🐛 问题描述

![67fe042234c007a571b277afd75b63e](https://github.com/lobehub/lobe-chat/assets/169867372/866ffcd0-c2a8-473c-8c45-4a4e9fe30b63)

### 📷 复现步骤

![67fe042234c007a571b277afd75b63e](https://github.com/lobehub/lobe-chat/assets/169867372/075c1c77-53ee-4206-8d27-8e6ed81382f8)

### 🚦 期望结果

![67fe042234c007a571b277afd75b63e](https://github.com/lobehub/lobe-chat/assets/169867372/e5577954-447c-4435-bb7b-85d1b39f051e)

### 📝 补充信息

![67fe042234c007a571b277afd75b63e](https://github.com/lobehub/lobe-chat/assets/169867372/d49a3870-c98b-48b4-8124-0a1b58d9a52c)

[a comment by an automated account is not shown]

**Comment by RubuJam** [comment, marked as answer]
请使用海外的镜像站，由于政策原因，国内镜像站已全数覆没

It is recommended that users access the site via mirror sites located outside of mainland China. This is due to the fact that all mirror sites within mainland China have been blocked as a result of policy considerations.

**Comment by xingpingcn** [comment]
找一个cf worker反代或者vercel反代的

[a comment by an automated account is not shown]

**Comment by zhangleijun111** [comment]
> > 机器人检测到问题正文的语言不是英语，请自动翻译。👯👭🏻🧑‍🤝‍🧑👫🧑🏿‍🤝‍🧑🏻👩🏾‍🤝‍👨🏿👬🏿
> 
> 查找 cf worker 或 vercel 反向生成

有教程吗

[a comment by an automated account is not shown]

**Comment by zhangleijun111** [comment]
> 有教程吗

有教程吗

[a comment by an automated account is not shown]

**Comment by xingpingcn** [comment]
> > 有教程吗
> 
> 有教程吗

自己找

[a comment by an automated account is not shown]

**Comment by zhangleijun111** [comment]
> 请使用海外的镜像站，由于政策原因，国内镜像站已全数覆没
> 
> 建议用户通过位于中国大陆以外的镜像站点访问该网站。这是因为由于政策考虑，中国大陆境内的所有镜像站点都被封锁了。

有网址吗

**Reply by Matinal111** [reply]
www.baidu.com

[a comment by an automated account is not shown]
